f/ansible-playbook.json

```
@@ -656,7 +656,7 @@
         },
         "changed_when": {
           "title": "Changed When",
-          "type": "boolean"
+          "type": "string"
```

This would prevent usage of an unconditional boolean value.

For both `failed_when` and `changed_when`, Ansible [accepts booleans and conditional expressions](https://docs.ansible.com/ansible/latest/reference_appendices/playbooks_keywords.html#task).

For `failed_when`, the schema already allows both.
The schema should allow both for `changed_when`, too.

```suggestion
          "type": ["string", "boolean"]
```